core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx

```diff
@@ -45,14 +45,13 @@ const OrderSummaryInfo = async ({
   const locale = await getLocale();
   const t = await getTranslations({ locale, namespace: 'Account.Orders' });
   const format = await getFormatter();
-  const { subtotal, shipping, tax, discounts, grandTotal } = summaryInfo;
-  const totalDiscountSum = discounts.couponDiscounts.reduce((sum, discount) => {
-    let totalDiscount = sum;
-
-    totalDiscount += discount.discountedAmount.value;
-
-    return totalDiscount;
-  }, 0);
+  const {
+    subtotal,
+    shipping,
+    tax,
+    discounts: { nonCouponDiscountTotal, couponDiscounts },
+    grandTotal,
+  } = summaryInfo;
```

Comment:
🍹 Just to make the formatting prettier
```suggestion
  const { subtotal, shipping, tax, discounts, grandTotal } = summaryInfo;
  const { nonCouponDiscountTotal, couponDiscounts } = discounts;
```